src/main/java/jenkins/plugins/slack/StandardSlackService.java

```diff
@@ -45,41 +45,70 @@
     private String host = "slack.com";
     private String baseUrl;
     private String teamDomain;
-    private String token;
-    private String authTokenCredentialId;
+    private String token = null;
```

Comment:
null is default 😕

Reply:
Removed all useless null initializations.